backend/cfg/cfg_dominators.ml

```diff
@@ -325,14 +328,21 @@ let compute_dominance_frontiers (cfg : Cfg.t) (doms : doms) :
       | _ :: _ :: _ ->
         List.iter predecessor_labels ~f:(fun predecessor_label ->
             let runner = ref predecessor_label in
-            while not (Label.equal !runner (Label.Tbl.find doms label)) do
+            let continue = ref true in
+            while
+              (not (Label.equal !runner (Label.Tbl.find doms label)))
```

Comment:
Why are we not using `_idom` instead of `Label.Tbl.find doms label` here?  (I know it's not related to the fix in this PR.)